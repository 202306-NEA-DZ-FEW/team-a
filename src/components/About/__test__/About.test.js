import renderer from "react-test-renderer";

import AboutPageCover from "..";

it("renders correctly", () => {
  const mockT = jest.fn();
  const tree = renderer.create(<AboutPageCover t={mockT} />).toJSON();
  expect(tree).toMatchSnapshot();
});
