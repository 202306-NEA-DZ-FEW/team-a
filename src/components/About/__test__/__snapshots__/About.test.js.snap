// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="hero min-h-screen"
  style={
    Object {
      "backgroundImage": "url(https://images.unsplash.com/photo-1469571486292-0ba58a3f068b?auto=format&fit=crop&q=80&w=1470&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D)",
    }
  }
>
  <div
    className="hero-overlay bg-opacity-50"
  />
  <div
    className="hero-content text-center text-neutral-content"
  >
    <div
      className="max-w-lg"
    >
      <h1
        className="mb-5 text-5xl font-bold"
      />
      <p
        className="mb-5"
      >
         
      </p>
    </div>
  </div>
</div>
`;
